### ISSUE TITLE
Move front-end bootstrap to tailwind

### ISSUE THREAD

**Issue by awalsh3** [Contributees/First-Ruby-Quest#29]
Views:

- [ ] ./app/views/shared/_navbar.html.erb - thank you @ClaireDMT! 
- [ ] ./app/views/shared/_cards.html.erb
- [ ] ./app/views/shared/_flashes.html.erb
- [ ] ./app/views/shared/_avatar.html.erb

- [ ] ./app/views/issues/index.html.erb - in progress 
- [ ] ./app/views/issues/show.html.erb
- [ ] ./app/views/issues/_form.html.erb
- [ ] ./app/views/issues/new.html.erb

- [ ] ./app/views/pages/dashboard.html.erb

**Comment by ClaireDMT** [comment]
I think that's a perfect! thanks for drafting a plan for this transition.

**Comment by Apurv428** [comment]
Hello @awalsh3, I'm keen on contributing to this issue. Could I take on this task? Additionally, is there a guide available for setting up the project?

**Comment by awalsh3** [comment]
Hey @Apurv428 definitely go for it! In terms of a guide… @ClaireDMT or @Oli0li might you have insight on this? I can’t remember what I did to setup the project 😅

**Comment by Apurv428** [comment]
@awalsh3 , Should I generate an issue for the specific file I intend to modify?

**Comment by awalsh3** [comment]
Yes, that would be great, thanks! On 11 Feb 2024, at 13:40, Apurv428 ***@***.***> wrote:﻿
@awalsh3 , Should I generate an issue for the specific file I intend to modify?

**Comment by Apurv428** [comment]
Okay, thank you!
